src/main/webapp/assets/modules/common/securityService.js

```diff
@@ -130,7 +151,7 @@ techlooper.factory("securityService", function (apiService, $rootScope, $q, util
           case jsonValue.views.employerDashboard:
             var lastPage = "/employer-dashboard";
           case jsonValue.views.createEvent:
-            var lastPage = "/create-event";
+            var lastPage = "/post-event";
```

Comment:
'lastPage' is already defined.<br>Expected a 'break' statement before 'case'.<br>'lastPage' is defined but never used.